examples/pytorch/logging/pytorch_log_model.ipynb

```diff
@@ -0,0 +1,440 @@
+    "            curr_loss = loss(outputs, labels)\n",
+    "            curr_loss.backward()\n",
+    "\n",
+    "            total_loss += curr_loss.item()\n",
+    "\n",
+    "            # Calculate accuracy using torchmetrics\n",
+    "            _, preds = torch.max(outputs, 1)\n",
+    "            total_accuracy += accuracy_metric(preds, labels).item()\n",
+    "\n",
+    "        avg_loss = total_loss / len(train_loader)\n",
+    "        avg_accuracy = total_accuracy / len(train_loader)\n",
+    "\n",
+    "        print(f\"Epoch {epoch + 1}, Loss: {avg_loss}, Accuracy: {avg_accuracy}\")\n",
+    "        mlflow.log_metric(\"loss\", avg_loss, step=epoch)\n",
+    "        mlflow.log_metric(\"accuracy\", avg_accuracy, step=epoch)\n",
+    "\n",
+    "    # Log the PyTorch model with the signature\n",
+    "    mlflow.pytorch.log_model(model, \"model\", signature=signature)\n",
+    "\n",
+    "    # Log parameters\n",
+    "    mlflow.log_param(\"epochs\", 10)\n",
+    "    mlflow.log_param(\"batch_size\", 16)\n",
+    "    mlflow.log_param(\"learning_rate\", 0.001)\n",
+    "\n",
+    "print(\"Model training and logging complete.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Loading the logged model back into memory"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "caac8221-e7bb-4772-9946-acd4c4f331e6",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Run id from the run above: e6f9b749409546758fba3ad9446f71ba\n",
+      "Original model output: tensor([[-0.0688, -0.3263, -0.0642]])\n",
+      "Loaded model output: tensor([[-0.0688, -0.3263, -0.0642]])\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Run id from the run above:\", run.info.run_id)\n",
+    "\n",
+    "# Later, or in a different script, you can load the model using the run ID\n",
+    "loaded_model = mlflow.pytorch.load_model(f\"runs:/{run.info.run_id}/model\")\n",
+    "\n",
+    "# you can now use the loaded model as you would've used the original pytorch model!\n",
+    "loaded_model.eval()\n",
+    "with torch.no_grad():\n",
+    "    sample_input = X_test_tensor[:1]\n",
+    "    loaded_output = loaded_model(sample_input)\n",
+    "    og_output = model(sample_input)\n",
+    "    print(\"Original model output:\", og_output)\n",
+    "    print(\"Loaded model output:\", loaded_output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# What you see in the mlflow UI\n",
+    "Please follow [this section of the contributing guide](https://github.com/mlflow/mlflow/blob/master/CONTRIBUTING.md#javascript-and-ui) to get the UI set up.\n",
```

Comment:
We will want to avoid cross reference and stick with inline instruction as much as possible. Cross reference creates a reading hassle.

Here let's just describe how to start a local MLflow server, and for detailed instruction on different types of tracking servers, we can refer to https://mlflow.org/docs/latest/getting-started/tracking-server-overview/index.html

Reply:
I think it's necessary, as just telling them to run `mlflow ui` might not be helpful enough. Moving this section to the beginning of the notebook though